src/main/groovy/com/palantir/gradle/javadist/tasks/CreateManifestTask.groovy

```diff
@@ -43,18 +43,24 @@ class CreateManifestTask extends BaseTask {
         return String.valueOf(project.version)
     }
 
+    @Input
+    public Map<String, Object> getExtraProperties() {
+        return Collections.emptyMap();
+    }
+
     @OutputFile
     File getManifestFile() {
         return new File("${project.buildDir}/deployment/manifest.yml")
     }
 
     @TaskAction
     void createManifest() {
-        EmitFiles.replaceVars(
-                CreateManifestTask.class.getResourceAsStream('/manifest.yml'),
-                getManifestFile().toPath(),
-                ['@serviceGroup@'  : getServiceGroup(),
-                 '@serviceName@'   : getServiceName(),
-                 '@serviceVersion@': getProjectVersion()])
+        getManifestFile().setText(JsonOutput.prettyPrint(JsonOutput.toJson(getExtraProperties() + [
```

Comment:
do we want to group the extra properties under a single key such as `ext`, `extensions`, `extras` or similar?